### PR TITLE
[5.2] Stdlib doc changes

### PR DESCRIPTION
**Pull request by amartini51**
Corrections to documentation comments.

Resolves rdar://problem/46837427&58855960&58717942

### CODE REVIEW

**Comment by compnerd** [comment]
I think that these should go into master and then get cherry-picked for the 5.2 release.

**Comment by amartini51** [comment]
@compnerd See also the [PR](https://github.com/apple/swift/pull/29436) to incorporate the same changes into 'master'.

**Comment by compnerd** [comment]
Normally, the changes are cherry-picked from master to indicate in history where the movement of the changes.  I don't think that its absolutely critical here, just perhaps something to consider in the future :)

**Comment by compnerd** [comment]
@swift-ci please smoke test

**Comment by compnerd** [comment]
@swift-ci please smoke test macOS platform

**Comment by amartini51** [comment]
Smoke test looks like it failed because of a SourceKit test that hardcodes the result of parsing some documentation comments into its test expectation.  Trying to reproduce locally so I can update the expectation.

**Review by airspeedswift — APPROVED** [review]

File: stdlib/public/core/OptionSet.swift [hunk 1]
```diff
@@ -278,7 +278,7 @@ extension OptionSet where Element == Self {
   ///     print(expressOption == .express)
   ///     // Prints "false"
   ///     print(expressOption == .secondDay)
-  ///     // Prints "true"
```
Comment (lorentey) [inline stdlib/public/core/OptionSet.swift]: This is indeed how OptionSet.remove works in 5.2, but that's actually a bug -- see https://github.com/apple/swift/pull/29436#discussion_r370906914

**Comment by futurejones** [comment]
Can one of the admins verify this patch?

File: stdlib/public/core/OptionSet.swift (hunk 1 above)
Comment (amartini51) [reply stdlib/public/core/OptionSet.swift]: Removed this change/commit on the PR against 'master'.  Will close this one.

**Pull request closed without merging**